KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/HighlightedText.java

```diff
@@ -406,12 +426,40 @@ public String getText() {
                 if (responseHighlightID == null) {
                     highlightedContent = attemptManualHighlighting(response.getResults(), highlightField, keywords);
                 } else {
-                    List<String> contentHighlights = responseHighlightID.get(LuceneQuery.HIGHLIGHT_FIELD);
-                    if (contentHighlights == null) {
-                        highlightedContent = attemptManualHighlighting(response.getResults(), highlightField, keywords);
+                    SolrDocument document = response.getResults().get(0);
```

Comment:
It might make it a little easier to read this method if this block of code related to mini chunk highlighting was factored out into its own method.